_summaries/salimans2016weight_2.md

```diff
@@ -1,19 +1,64 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{salimans2016weight}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: samatharhay
+score: 9
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+The paper introduces weight normalization, a reparameterization of the weight vectors of a neural network, to speed up convergence of stocastic gradient descent.
+
+In standard neural networks the weighted sum computed by each neuron is as follows:
+**intsert equation 1**
+where w (weights) and x (input) are both k-dimensional vectors and b is a scalar bias term.
+
+In weight normalization each w vector is reparameterized in terms of a parameter vector v, of dimensionality k, and a scalar parameter g: 
+**insert equation 2**
+
+Since the mean of the neuron activations are still dependent on v a varient of batch normalization, called mean-only batch normalization, was introduced.
+
+Similarly to to full batch normalization, the minibatch means are subtracted out, but the minibatch standard deviations are not divided by when computing neuron activations.  The activations are computed as follows:
+**insert equation 7**
+
+The gradient of the loss with respect to preactivation t is:
+**insert equation 8**
+
+The prove the usefulness of their model the authors conducted experiments using 4 different models.
+
+ Supervied Classification: CIFAR-10
+* Model based off of the ConvPool-CNN-C architecture, with the modifications of replacing the first drop out layer with a layer that adds Gaussian noise, expanding the last hidden layer from 10 to 192 units and using a 2X2 max-pooling layer instead of a 3X3.
+* Batch nomalization preformed 16% slower than the baseline, while weight normalization was not noticable slower
+* Mean-only batch normalization with weight normalization preformed the best (7.31% error)
```

Comment:
* preformed --> performed

Might be a good idea to mention the hypothesis that mean-only B.N. with weight normalization is helpful because batch normalization may have introduced noise that are useful for regularizing the network while weight normalization.